lib/currency.spec.js

```diff
@@ -111,12 +137,13 @@ describe('Currency', () => {
         }
     });
 
-    context('Factory return null on illegal input', () => {
-        for (const [ct, id] of badConstructorArgs) {
-            it (`with "${ct}" ${typeStr(ct)}, "${id}" ${typeStr(id)}`, () => {
-                expect(Currency.from({ct, id})).to.equal(null);
+    context('Factory returns null on illegal input', () => {
```

Comment:
This test sounds inverted (context should define the scenario, inputs, etc, *not* the expected result). Not a showstopper but something to keep in mind for the next interation.

Reply:
I think that makes a lot of sense!